sdk/network/Azure.ResourceManager.Network/src/Generated/FirewallPolicyRuleCollectionGroupDraftData.cs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using Azure.Core;
+using Azure.ResourceManager.Network.Models;
+
+namespace Azure.ResourceManager.Network
+{
+    /// <summary>
+    /// A class representing the FirewallPolicyRuleCollectionGroupDraft data model.
+    /// Rule Collection Group resource.
+    /// </summary>
+    public partial class FirewallPolicyRuleCollectionGroupDraftData : NetworkResourceData
+    {
+        /// <summary> Initializes a new instance of <see cref="FirewallPolicyRuleCollectionGroupDraftData"/>. </summary>
+        public FirewallPolicyRuleCollectionGroupDraftData()
+        {
+            RuleCollections = new ChangeTrackingList<FirewallPolicyRuleCollectionInfo>();
+        }
+
+        /// <summary> Initializes a new instance of <see cref="FirewallPolicyRuleCollectionGroupDraftData"/>. </summary>
+        /// <param name="id"> Resource ID. </param>
+        /// <param name="name"> Resource name. </param>
+        /// <param name="resourceType"> Resource type. </param>
+        /// <param name="serializedAdditionalRawData"> Keeps track of any properties unknown to the library. </param>
+        /// <param name="size"> A read-only string that represents the size of the FirewallPolicyRuleCollectionGroupProperties in MB. (ex 1.2MB). </param>
+        /// <param name="priority"> Priority of the Firewall Policy Rule Collection Group resource. </param>
+        /// <param name="ruleCollections">
+        /// Group of Firewall Policy rule collections.
+        /// Please note <see cref="FirewallPolicyRuleCollectionInfo"/> is the base class. According to the scenario, a derived class of the base class might need to be assigned here, or this property needs to be casted to one of the possible derived classes.
+        /// The available derived classes include <see cref="FirewallPolicyFilterRuleCollectionInfo"/> and <see cref="FirewallPolicyNatRuleCollectionInfo"/>.
+        /// </param>
+        internal FirewallPolicyRuleCollectionGroupDraftData(ResourceIdentifier id, string name, ResourceType? resourceType, IDictionary<string, BinaryData> serializedAdditionalRawData, string size, int? priority, IList<FirewallPolicyRuleCollectionInfo> ruleCollections) : base(id, name, resourceType, serializedAdditionalRawData)
+        {
+            Size = size;
+            Priority = priority;
+            RuleCollections = ruleCollections;
+        }
+
+        /// <summary> A read-only string that represents the size of the FirewallPolicyRuleCollectionGroupProperties in MB. (ex 1.2MB). </summary>
+        public string Size { get; }
```

Comment:
Is this really a string type?

Reply:
Yes, it has "MB" at the end . It is actually the "file size"